<launch>
   <param name="use_sim_time" value="true"/>
   <node pkg="rosbag" type="play" name="player" output="screen" args="--clock /home/tx/lanelet_costmap_ws/src/lanelet_to_costmap/map_file/data/bag/with_road.bag"/>

   <node pkg="pcl_test" type="points_filter" name="points_filter" output="screen" >
      <param name="min_x" value="-1.6"/>
      <param name="min_y" value="-0.6"/>
      <param name="min_z" value="-100"/>

      <param name="max_x" value="0.6"/>
      <param name="max_y" value="0.6"/>
      <param name="max_z" value="100"/>

      <param name="points_topic" type="string" value="/rslidar_points"/>
      <param name="filter_topic" type="string"  value="filter_points"/>

   </node>

   <node pkg="rviz" type="rviz" name="rviz" args="-d $(find pcl_test)/filter.rviz"/>

</launch>
